mscore/plugin/pluginUpdater.cpp

```diff
@@ -864,20 +865,21 @@ void PluginWorker::updateInstall(QPushButton* button)
             }
       button->setText(tr("Downloading..."));
       QString localPath = download(page_url, true);
-      if (install(localPath)) {
+      PluginStatus res;
+      if (install(localPath), &res) {
```

Comment:
This is a very obscure form of an if statement, using the comma operator...
and I have no idea why you used that?

Reply:
Ah, actually it should be `if (install(localPath, &res)) {`

Reply:
Ah, it's an error and the `&res` should have been in the parentheses... Already fixed that.